### DISCUSSION TITLE
Line width problem

### DISCUSSION THREAD

**Discussion by zsoooc** [visgl/deck.gl#8280] [Q&A]
I use MapView with zoom level = 21, pitch = 70 and I would like to set the width of the (MVTLayer) line geometry precisely. I would like to set to 8 meters. However, I recognized that the line width does not seem to change, The line width appears to be exactly the same if I set bigger lineWidth value.  
For example, look a these screenshots. It is the same linestring with the same zoom, pitch and camera position. 
First image: the line width is 1 m. Second: the line width is 8 m. The line width appears to be exactly the same. Why?

How can I set the line width more accurately?

This is props of the layer:
```

                    id: props.id + '1',
                    getPosition: d => d.geometry.coordinates,
                    getLineColor: d => [0, 0, 0],
                    lineWidthUnits: 'meters',
                    lineWidthScale: 1,
                    getlineWidth: 1
```
![image](https://github.com/visgl/deck.gl/assets/19270270/76db9578-4831-4b97-9ad2-ededf77c59c4)
![image](https://github.com/visgl/deck.gl/assets/19270270/83b1bbda-dd03-42d2-8598-406256bb014c)

**Comment by tauseefmhta** [comment]
Which version you are using?

**Reply by zsoooc** [reply]
v8.9.35

**Comment by Pessimistress** [comment]
Are you setting `lineWidthMaxPixels`?

**Reply by zsoooc** [reply]
No, I didn't try set lineWidthMaxPixels.

**Reply by Pessimistress** [reply]
Can you make a Codepen that reproduces your issue?
